frontend/src/app/game/events-layout/events-layout.component.ts

```diff
@@ -27,7 +27,9 @@ export class EventsLayoutComponent {
       }
     }
 
-    this.gameService.event = undefined;
-    this.gameService.setSpeed('play');
+    this.gameService.currentEvent = this.gameService.events.shift();
```

Comment:
This is where we take another event from the event queue

Reply:
Sorry, I was searching by the hidden method name.